pom.xml.bak

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file should not be in the git. put it in gitignore and delete this file.

Reply:
I have to delete this file in Eclipse IDE.

Reply:
completed be4a6f07f5b009f156251f7cd647330ed1d285f1